scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -15,106 +16,105 @@ object ScaladocParser {
 
   private val numberOfSupportedHeadingLevels = 6
 
-  private val hspaceChars = "\t\r "
```

Comment:
I inlined it, because otherwise it keeps failing with:
```
[error] /home/tgodzik/Documents/scalameta/scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala:20:58: Function can only accept constant singleton type
[error]   private def hspacesMin[_: P](min: Int) = P(CharsWhileIn(hspaceChars, min))
[error]                                                          ^
[error] one error found
[error] (parsersJVM / Compile / compileIncremental) Compilation failed
[error] Total time: 1 s, completed Mar 2, 2021 4:56:29 PM
```